source/_posts/2021-12-21-android-december-2021-release.markdown

```diff
@@ -0,0 +1,129 @@
+---
+title: "Android 2021.12: Wear OS Beta!"
+description: "Wear OS Beta, Instant Update Widgets and more!"
+date: 2021-12-19 00:00:00
+date_formatted: "December 19, 2021"
+comments: true
+author: Daniel Shokouhi
+categories: Release-Notes
+og_image: /images/blog/2021-12-21-android-december-2021/Companion.png
+---
+
+
+![Screenshot of the Android app](/images/blog/2021-12-21-android-december-2021/Companion.png)
+
+Hey Everyone! It's time for the December 2021 Android release. It has been a while since the last Android release as the team has been very busy working on many new and exciting features. To kick things off we would like to announce that there is now a Wear OS app that you can find in the Play Store alongside todays phone app release!
+
+## Wear OS Beta
+
+For the past couple of months the Android repo has been seeing a lot new contributors coming and bringing in some amazing work. There is now a Wear OS Beta app released in the Play Store! A big thank you to [leroyboerefijn], [dshokouhi], [JBassett], [Kisty], [apo-mak], 
+[SketchyWolf] and [HunterX86] for all your hard work! A lot of work has been done to share the codebase between the phone and the watch because we wanted the watch to also have a standalone experience in case you are not near your phone. The app will remain as a Beta for several months but we feel in its current state it is ready for you to enjoy. The reason we have decided to keep it with a beta label for now is because there is more work to be done and some of the underlying libraries being used have not received a stable release yet.
```

Comment:
Another nitpick name is also spelled wrong here

Reply:
thought i hit save before the last commit lol oops